.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
```

Comment:
IMHO this file should belong ideally under the directory `workflows/`

and the other file of the same name in general either renamed or removed (here and mostly probably removed)